#180 - increasing screen size to default to 1280x1024

beenanner opened the pull request:
Will update the changelog once we've confirmed this is working as expected.

tobli commented on lib/support/startVideo.js:
```diff
@@ -7,7 +7,7 @@ const webdriver = require('selenium-webdriver'),
 function getFFMpegArgs(mpegPath, options) {
   // calculate the viewport, lets spend some time and make this better
-  let size = '1366x708';
+  let size = '1280x1024';
```
This should be 1280x964, right? Shaving off 60px for browser ui (window chrome). Could be cleaner to do
```
const size = options.viewPort || '1280x1024';
```
and drop the if-statement to shave off the 60px just in one place.

beenanner replied:
oops forgot all about this I can make those changes @tobli :-)

tobli commented:
@beenanner is this relevant now that #180 is solved? My earlier comment about the if-statement was unclear. I meant removing just the if-statement, not the body. I.e. unconditionally parse remove 60 vertical pixels, either from the viewPort option, or from '1280x1024'.

beenanner commented:
@tobli I believe this is still relevant. I went ahead and reverted my previous commits that were incorrect and added back 60 pixel removal. Comparing the video produced by master vs this branch it seems better for Firefox. Chrome still seem a bit off centered though. Feel free to test this branch out if you have time.

beenanner commented:
Don't believe this is relevant anymore :-)
